modules/installation-initializing.adoc

```diff
@@ -125,9 +133,9 @@ Installing OpenShift Container Platform (OCP) version 4.6 on Red Hat Virtualizat
 ====
 endif::openshift-origin[]
 endif::rhv[]
-ifdef::vsphere[]
+ifdef::vsphere,vmc[]
 VMware vSphere.
```

Comment:
This isn't showing in either of these places: 

https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations.html#installation-initializing_installing-vmc-network-customizations

https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-customizations.html#installation-initializing_installing-vmc-customizations